### ISSUE TITLE
css-loader 5.x.x support?

### ISSUE THREAD

**Issue by nolimitdev** [vuejs/vue-style-loader#50]
vue-style-loader does not work also with css-loader 5.x.x such as with [css-loader 4.x.x](https://github.com/vuejs/vue-style-loader/issues/46)

**Comment by Byur** [comment]
> vue-style-loader does not work also with css-loader 5.x.x such as with [css-loader 4.x.x](https://github.com/vuejs/vue-style-loader/issues/46)

just set value of  attribute "esModule" to false in css-loader's options.

**Comment by magicalstack** [comment]
Useful! 👍 
```
{
  test: /\.styl(us)?$/,
  use: [
    'vue-style-loader',
    {
      loader: 'css-loader',
      options: {
        esModule: false,
      },
    },
    'stylus-loader',
  ],
},
```

**Comment by nolimitdev** [comment]
I think that the best is to not use obsolete `vue-style-loader` at all and use rather original `style-loader`.

**Comment by shunyue1320** [comment]
### Pro-test effective

webpack.config.js：
```
{
  test: /\.css$/,
  use: [
    'vue-style-loader',
    {
      loader: 'css-loader',
      options: {
        esModule: false
      }
    }
  ]
},
```

package.json:
```
"css-loader": "3.6.0",
"vue-loader": "15.9.6",
```

**Comment by haoqunjiang** [comment]
Fixed in 4.1.3
